pkg/skaffold/event/proto/skaffold.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+package proto;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+message StateResponse {
+  State state = 1;
+}
+
+message Response {
+  string msg = 1;
+}
+
+message Request {
+  string name = 1;
+}
+
+message State {
+  BuildState buildState = 1;
+  DeployState deployState = 2;
+  map<string, PortInfo> forwardedPorts = 3;
+}
+
+// BuildState contains a map of all skaffold artifacts to their current build
+// states
+message BuildState {
+  map<string, string> artifacts = 1;
+}
+
+// DeployState contains the status of the current deploy
+message DeployState {
+  string status = 1;
+}
+
+enum EventType {
```

Comment:
1. instead of `EventType`  enum, we should model this with [union types](https://developers.google.com/protocol-buffers/docs/proto3#oneof). That way the _schema_ for each event type is documented nicely in the proto description. 

2. We also need more comments here.